browser/webpack.config.js

```diff
@@ -79,6 +79,15 @@ const config = {
           loader: '@gnomad/markdown-loader',
         },
       },
+      {
+        test: /robots\.txt$/,
+        use: {
+          loader: 'file-loader',
+          options: {
+            outputPath: 'assets/images',
```

Comment:
Does this `outputPath` parameter need to change? Googling suggests that a `robots.txt` file has to be at the root of the build output. 

Perhaps `'assets/images'` -> `'/'`